src/skmultiflow/trees/regression_hoeffding_adaptive_tree.py

```diff
@@ -1,16 +1,14 @@
-from abc import ABCMeta, abstractmethod
+import numpy as np
 
-from skmultiflow.trees.regression_hoeffding_tree import RegressionHoeffdingTree, HoeffdingTree
-from skmultiflow.utils.utils import *
-from skmultiflow.drift_detection.adwin import ADWIN
-from skmultiflow.utils import check_random_state
+from skmultiflow.trees import RegressionHoeffdingTree
+
+from skmultiflow.trees.nodes import AdaSplitNodeForRegression
+from skmultiflow.trees.nodes import AdaLearningNodeForRegression
 
 
 _TARGET_MEAN = 'mean'
 _PERCEPTRON = 'perceptron'
 error_width_threshold = 300
```

Comment:
Use uppercase for constants.

Reply:
Change addressed in commit [a8c8022](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/a8c8022987ea6ac45b8aea5fa0cc2a1ab2b20384).